.changeset/thin-buckets-juggle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-airbrake': minor
+'example-app': patch
```

Comment:
No need to include `example-app` in the changeset since it's not a published package

Reply:
This was auto-generated I believe. Should I manually just remove this line?

Reply:
Yep! It's from the changeset cli, and if you select "all changed packages" it'll be included. Got an open issue over there for a way around that but for now they simply need to be avoided

Reply:
Removed